docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -0,0 +1,87 @@
+# Release Automation
+
+## Status
+
+In Review for Geneva
+
+## Context
+
+EdgeX Foundry is a framework composed of microservices to ease development of IoT/Edge solutions. With the framework getting richer, project growth, the number of artefacts to be released has increased. This proposal outlines the automation of the release process for the base artifacts.
+
+## Requirements
+
+### Release Artifact Definition
+
+For the scope of Geneva release artifact types are defined as:
+
+- GitHub tags in the repositories.
+- Docker images in our Nexus repository and Docker hub.
+- Snaps in the Snapcraft store.
+
+This list is likely to expand in future releases.
+
+### General Requirements
+
+As the EdgeX Release Czar I gathered the following requirements for automating this part of the release.
+
+1. The release automation needs a manual trigger to be triggered by the EdgeX Release Czar or the Linux Foundation Release Engineers. The goal of this automation is to have a "push button" release mechanism to reduce human error in our release process.
```

Comment:
Lisa - with Andy G. from LF suggesting new self service releases - do we need to look at what he has before approving this?

Reply:
My opinion is that we should evaluate what they have when it is available. Essentially we are creating our own "self-service" release in Geneva. As a community we can figure out a migration plan if applicable. Self-service releases from Github aren't currently available from the LF but that was slotted for Q2/Q3. Also from the verbiage that the LF has given is that they just started the explore for all of this including the Jenkins pipelines.